Applied-Research-Winter-2022/Pozter
# [POZ-11] & [POZ-14]  Added table, Populate tables and update at triggers

## Link Jira ticket:
https://jess-nguy.atlassian.net/jira/software/projects/POZ/boards/3/backlog?selectedIssue=POZ-11
https://jess-nguy.atlassian.net/jira/software/projects/POZ/boards/3/backlog?selectedIssue=POZ-14

## Author Notes:
Created SQL for database schema (tables)
Create SQL for seeded inserts for billboards, owners, and screen_canvases

I added venue as a column for billboards because in Florin's Spec sheet it had a venue name.
All inserts were taken from Florin's spec sheet which is in the dropbox.

## Impacts and Notes:
This impacts the billboard endpoint. The endpoint should now show more details and can return specific rows or all billboards.

## Testing Guidance:
- Open Database and see all the tables created as seen in the files changes tab. See discord linked-resource channel for postgres login. You can use either pgAdmin or DBeaver to see the tables. https://dbeaver.io/download/ 
If you use DBeaver: 
![image](https://user-images.githubusercontent.com/38814279/155265125-ad9a2f3e-11ac-4ad9-a627-6d6e8fb07254.png)
In DBeaver to see all tables open `Schemas -> Public -> Tables`
- Ensure the endpoint still works. See the testing section for instruction in this PR  https://github.com/Applied-Research-Winter-2022/Pozter/pull/2

## Author Checklist:

- [x] Change Jira ticket status to into Review.
- [x] Updated the documentation. **No updates to docs needed**
